.github/workflows/auth-dependabot.yml

```diff
@@ -0,0 +1,24 @@
+name: auth dependabot CI
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  dependabot-comments:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.15.4"
+      - run: yarn install
```

Comment:
Follow up to [this thread](https://github.com/marilyn79218/react-lazy-show/pull/58/files#r664377395), but don't forget to install dependencies before ahead.